### ISSUE TITLE
Hard Time with Layouts...

### ISSUE THREAD

**Issue by ArchV12** [arkayenro/arkinventory#1228]
What I want to do seems pretty straight forward, but doesn't appear to be working...

On one character I want a bar that had Leatherworking Stuff... and on another character I want a bar that stores Mining Stuff (basically replacing the LW bar).

For the Miner Profile I have it set to "Main" style and "Miner" layout.
For the LW Profile I have it set to "Main" style and "Leather" layout.

In theory these two should have the different bars.  However if I edit Leather layout and change the color or categories, it also changes the Miner layout.  

Basically it appears that there's only ever ONE layout.

Am I missing something?

**Comment by ArchV12** [comment]
NEVERMIND!

I totally missed the "Current Profile" drop down at the top under the General Tab.  For some reason I thought I was changing profiles on the Profile tab. :)
